server/plugin.go

```diff
@@ -64,7 +65,16 @@ func (p *Plugin) OnActivate() error {
 	stored := api.StoredConfig{}
 	_ = p.mm.Configuration.LoadPluginConfiguration(&stored)
 
-	awsClient := aws.NewAWSClient(stored.AWSAccessKeyID, stored.AWSSecretAccessKey, &mm.Log)
+	accessKey := os.Getenv("APPS_INVOKE_AWS_ACCESS_KEY")
+	if accessKey == "" {
+		mm.Log.Warn("APPS_INVOKE_AWS_ACCESS_KEY is not set. AWS apps won't work.")
+	}
+	secretKey := os.Getenv("APPS_INVOKE_AWS_SECRET_KEY")
+	if secretKey == "" {
+		mm.Log.Warn("APPS_INVOKE_AWS_SECRET_KEY is not set. AWS apps won't work.")
+	}
+
+	awsClient := aws.NewAWSClient(accessKey, secretKey, &mm.Log)
```

Comment:
if `APPS_INVOKE_AWS_ACCESS_KEY` and `APPS_INVOKE_AWS_SECRET_KEY` are empty `aws.NewAWSClient` may still create a credentials. It is reading from the "default env vars" `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. So either we get rid of the previous behaviour or correct the messages or get rid of this new env vars.

Reply:
I'm inclined to get rid of get rid of the previous behavior and don't use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. Given that we are using two sets of credentials, I think it's better to be explicit. @iomodo What do you think?

Reply:
Agree

Reply:
Changed via c37e9d8